sklearn/feature_selection/rfe.py

```diff
@@ -102,7 +103,13 @@ class RFE(BaseEstimator, MetaEstimatorMixin, SelectorMixin):
            Mach. Learn., 46(1-3), 389--422, 2002.
     """
     def __init__(self, estimator, n_features_to_select=None, step=1,
-                 estimator_params={}, verbose=0):
+                 estimator_params=None, verbose=0):
+        if estimator_params is not None:
+            warnings.warn("The 'estimator parameters' is deprecated as of version 0.16 "
```

Comment:
You should use the actual name ``estimator_params`` and either say Just "``estimator_params is deprecated" or "The parameter ``estimator_params``" to make a complete english sentence.